concepts/prompt.mdx

```diff
@@ -2,15 +2,79 @@
+  <img src="/images/playground-prompt-template.png" alt="An example of a prompt template in the playground." />
+</Frame>
+
+### 1. Design prompt templates
+
+On the left hand side of the view you can create prompt templates (1). 
+* In a `system message` you give instructions about the role of the assistant. For example, you can say that the assistant should provide concise answers. 
+* In `User`, you give a format of the user query to the LLM. In this example, two variables are used. First, context documents are given, in a for-loop. The content of the variables can be defined below, but are usually defined in the code of the application. Then, the question that the user entered in the chatbot window is given.
+* `Assistant` messages are responses by the LLM. 
+* `Tool` messages are responses by the tool or function. 
+
+You can also pass functions to the prompt template. They are defined in the code or in `Tools`. The LLM model can detect when a specific function needs to be called based on the user's input. 
+
+### 2. LLM settings
+
+On the right side of the screen (2), you can set an LLM provider (for example OpenAI or Anthropic). You can change the following settings: 
+* **Model.** Which model of the provider to use.
+* **Temperature.** Control randomness. A lower temperature results in less random generations. As the temperature approaches zero, the model will become deterministic and repetitive. 
+* **Maximum Length.** Maximum length of tokens to generate. Requests can use op to 32,000 tokens, shared between prompt and completion. The exact limit varies per model.
+* **Stop Sequences.** Use up to four sequences where the API will stop generating further tokens. The returned text will not contain the stop sequence.
+* **Top P** or Nucleas Sampling. Controlling diversity by how many possible words to consider. A high Top P will look at more possible words, even the less likely ones, which makes the generated text more diverse.
+* **Frequency Penalty.** How much to penalize new tokes based on their existing frequency in the text so far. Decreases the model's likelihood to repeat the sam line verbatim.
+* **Presence Penalty.** How much to penalize new tokens based on whether they appear in the text so far. Increases the model's likelihood to talk about new topics.
+
+### 3. Try out & in-context debugging.
+
+In the center of the screen (3), you can try out the settings and prompt template. Make sure you have an API key of the selected provided set (at the top right key icon). Then, you can send messages to the LLM using this template and settings. You can change things, and try again. 
+
+If you accessed the playground via an LLM call, you get the conversation context which you can use to try out different settings. 
+
+You can also use multimodality here. For example, you can upload an image in the chat. 
+
+
+### 4. Save
+
+Once you are happy with the template and settings, you can save the setup (4). 
```

Comment:
Add visual and explanation on diff and changelog
<img width="666" alt="image" src="https://github.com/Chainlit/literal-docs/assets/16107237/5c0538ae-6a73-46ee-835d-02d93ff04e69">